### PR TITLE
Features/prism

### PR DESCRIPTION
**Pull request by staccDOTsol**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
 	},
 	"scripts": {
 		"wizard": "node ./src/index.js",
-		"start": "node ./src/index.js && node ./src/bot/index.js",
+		"start": "cp -r buffer-layout/* node_modules/@solana/buffer-layout/ node ./src/index.js && node ./src/bot/index.js",
```
Comment (alexonab) [inline package.json:15]: This needs && between cp and node.

`"start": "cp -r buffer-layout/* node_modules/@solana/buffer-layout/ && node ./src/index.js && node ./src/bot/index.js",

File: src/bot/setup.js [hunk 2]
```diff
@@ -136,7 +145,7 @@ const getInitialoutAmount = async (
 			inputMint: new PublicKey(inputToken.address),
 			outputMint: new PublicKey(outputToken.address),
 			amount: amountToTrade,
```
Comment (alexonab) [inline src/bot/setup.js:147]: jup-ag v4 requires a BigInt for the amount.

File: package.json (hunk 1 above)
Comment (staccDOTsol) [reply package.json:15]: good call, I eyeballed it <3

Comment (staccDOTsol) [reply package.json:15]: @arbcaptain can you approve, I cannot

File: src/bot/setup.js (hunk 2 above)
Comment (staccDOTsol) [reply src/bot/setup.js:146]: @arbcaptain @alexonab I don't see the change, but if it's here, plz approve

**Pull request merged**